### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by jgoodknight**
Project import generated by Copybara.

### CODE REVIEW

File: src/common/noisers.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    return np.round(self._func_with_noise(x))
+
+
+def main(argv):
+  if len(argv) > 1:
+    raise app.UsageError('Too many command-line arguments.')
+
+
+if __name__ == '__main__':
+  app.run(main)
```
Comment (pfhgetty) [inline src/common/noisers.py:114]: Does this really run? Where is app imported?

File: src/estimators/estimator_noisers.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      return self._noiser(cardinality_estimate)
+
+
+def main(argv):
+  if len(argv) > 1:
+    raise app.UsageError('Too many command-line arguments.')
+
+
+if __name__ == '__main__':
+  app.run(main)
```
Comment (pfhgetty) [inline src/estimators/estimator_noisers.py:74]: Same for this one

File: src/estimators/tests/bloom_filters_test.py [hunk 3]
```diff
@@ -120,6 +120,12 @@ def test_multi_insertion_same_element(self):
     b1.add(1)
     self.assertEqual(UnionEstimator.estimate_cardinality(b1), 1)
 
+  def test_raise_error_with_full_bloom_filter(self):
+    b1 = BloomFilter(length=2, random_seed=3)
+    b1.add_ids(range(10))
+    with self.assertRaises(ValueError):
+      _ = UnionEstimator.estimate_cardinality(b1)
```
Comment (pfhgetty) [inline src/estimators/tests/bloom_filters_test.py:127]: This is confusing syntax. Why are we assigning a throwaway to the return of this function? Shouldn't we just call the function instead without assigning anything?

File: src/common/noisers.py (hunk 1 above)
Comment (jgoodknight) [reply src/common/noisers.py:114]: yeesh, good catch.  Normally we would do 'from absl import app' but this doesn't need to be a executable anyway

File: src/estimators/estimator_noisers.py (hunk 2 above)
Comment (jgoodknight) [reply src/estimators/estimator_noisers.py:74]: done

File: src/estimators/tests/bloom_filters_test.py (hunk 3 above)
Comment (jgoodknight) [reply src/estimators/tests/bloom_filters_test.py:127]: another good call, will remove

**Comment by jgoodknight** [comment]
I think we messed something up internally, my apologies.  Will have to regenerate this PR

**Pull request closed without merging**